flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -960,11 +997,12 @@ protected Collection<RowData> convertToRowData(DataStructureConverter converter)
                     final RowData rowData = (RowData) converter.toInternal(row);
                     if (rowData != null) {
                         rowData.setRowKind(row.getKind());
-                        result.add(rowData);
+                        partitionResult.add(rowData);
+                        size++;
```

Comment:
It seems size == partitionResult.size() ? Could we directly use partitionResult.size() ?

Reply:
The `partitionResult.size()`  is the size of a specific partition, while the `size` is all accepted partitions total size. There is a for-loop at line 961